examples/SSVEP/README.txt

```diff
@@ -0,0 +1,4 @@
+Classification of SSVEP
+---------------
```

Comment:
line is to short
```suggestion
-----------------------
```
it should match the number of characters above otherwise it's an RST bug

Reply:
Thank you, I thought a minimal number of dashes was enough!